index.bs

```diff
@@ -2883,8 +2883,8 @@ When registering a new credential, represented by an {{AuthenticatorAttestationR
 1. If [=user verification=] is required for this registration, verify that the [=User Verified=] bit of the <code>[=flags=]</code>
     in |authData| is set.
 
-1. If [=user verification=] is not required for this registration, verify that the [=User Present=] bit of the
-    <code>[=flags=]</code> in |authData| is set.
+1. If the [=User Verified=] bit of the <code>[=flags=]</code> in |authData| is not set, verify that the [=User Present=] bit of
```

Comment:
When will user presence bit not set? As of now, silent signatures are not allowed in WebAuthN. I would remove "If the [=User Verified=] bit of the <code>[=flags=]</code> in |authData| is not set" all together.

Reply:
The User _Present_ bit will always be set, but the UV bit might not.

Reply:
I would like to do that, but the CTAP drafts I'd read at the time didn't make it clear that UP would always be required. It looks like it is in the latest drafts, though. Thanks!